Doc/library/json.rst

```diff
@@ -349,8 +352,9 @@ Basic Usage
 
    The other arguments have the same meaning as in :func:`load`.
 
-   If the data being deserialized is not a valid JSON document, a
-   :exc:`JSONDecodeError` will be raised.
+   If the data is not a valid JSON document, a :exc:`JSONDecodeError` will be
+   raised. If the binary file does not contain UTF-8, UTF-16 or UTF-32 encoded
+   data, a :exc:`UnicodeDecodeError` will be raised.
```

Comment:
Let's just remove this; there is no need to duplicate what's in the parameter list nearby.

Reply:
Sorry, I did not clearly communicate what I meant. I put up an amendment PR: #128609.